packages/loot-core/src/platform/client/undo/index.d.ts

```diff
@@ -8,6 +8,7 @@ export type UndoState = {
     name: string;
     items: Set<string>;
   } | null;
+  current: unknown;
```

Comment:
Instead of typing as `unknown`, does it work to type it like this and avoid some type casts?

```typescript
type CurrentUndoState {
  id?: string;
  url: unknown;
  openModal: ModalType;
  selectedItems: {
    name: string;
    items: Set<string>;
  } | null;
}

export type UndoState = CurrentUndoState & {
  current: CurrentUndoState | null;
}
```